Simulator/simulator/plant_presets.py

```diff
@@ -29,17 +49,18 @@ def _compute_from_table_values(
     h_0 = 0.1
     r_0 = 0.1 / garden_ratio
     r_max = seed_spacing / 2 / garden_ratio
-    growth_time = int(maturation_time - (germination_time[0] + germination_time[1]) / 2)
-    c1 = (((r_max / r_0) ** (1 / growth_time) - 1) * STEP) / (k2 * c2 * (1.5 * np.pi) ** 0.5)
+    growth_time, c1, _ = generate_c1_and_growth_time(germination_time, maturation_time, r_max, r_0, k2, c2) 
```

Comment:
You sample some normal distributions in `generate_c1_and_growth_time()` hence `growth_time, c1, germination_length` will be different here vs simulator.plant_type.py, right? They shouldn't differ.

Reply:
We want each plant to have different values even if they're from the same plant type.  Otherwise all plants of the same type will have the exact same germination time, growth time, c1.